evm/src/cpu/kernel/tests/core/jumpdest_analysis.rs

```diff
@@ -37,10 +39,82 @@ fn test_jumpdest_analysis() -> Result<()> {
     ];
     let mut interpreter = Interpreter::new_with_kernel(jumpdest_analysis, initial_stack);
     interpreter.set_code(CONTEXT, code);
-    interpreter.set_jumpdest_bits(CONTEXT, jumpdest_bits);
+    interpreter.set_jumpdest_analysis_inputs(HashMap::from([(
+        3,
+        BTreeSet::from_iter(
+            jumpdest_bits
+                .iter()
+                .enumerate()
+                .filter(|&(_, &x)| x)
+                .map(|(i, _)| i),
+        ),
+    )]));
+
+    assert_eq!(
+        interpreter.generation_state.jumpdest_table,
+        Some(HashMap::from([(3, vec![0, 1, 0, 5, 0, 7])]))
```

Comment:
A comment to justify the `vec![0, 1, 0, 5, 0, 7]`? Also: can it be a constant perhaps?

Reply:
I also think some description may be beneficial (though it also applies to other tests in the suite that are a bit cryptic).